core/templates/pages/exploration-player-page/layout-directives/exploration-footer.component.html

```diff
@@ -57,6 +32,195 @@
+     */
+    top: auto !important;
+    transform: none !important;
+    will-change: auto !important;
+  }
+
+  .oppia-exploration-footer .author-profile-dropdown-menu li {
+    background: #fff;
+    color: #094142;
+    float: none;
+    padding: 15px 0;
+    text-overflow: clip;
+    white-space: nowrap;
+    width: 100%;
+  }
+
+  .oppia-exploration-footer .author-profile-dropdown-menu li a:hover {
+    background-color: #eee;
+    color: #888;
+  }
+
+  .oppia-exploration-footer .author-profile-text {
+    cursor: default;
+    font-size: 13px;
+    line-height: 1.5;
+    margin: 12px 0 17px 0;
+    padding-left: 6px;
+    text-overflow: clip;
+    white-space: nowrap;
+  }
+
+  .oppia-exploration-footer .author-profile-text.not-hovered {
+    color: #fff;
+    text-transform: uppercase;
+  }
+
+  .oppia-exploration-footer .author-profile-text.hovered-text,
+  .oppia-exploration-footer .author-profile .hover-link:focus .author-profile-text {
+    color: #094142;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  .oppia-exploration-footer .oppia-share-exploration-footer {
+    line-height: 0.6;
+    padding-right: 20px;
+    text-transform: uppercase;
+  }
+
+  .oppia-exploration-footer .oppia-exploration-footer-container {
+    max-width: 800px;
+  }
+
+  .oppia-exploration-footer .oppia-exploration-footer-sharing-links ul {
+    display: block;
+    float: right;
+    margin: -8px;
+    padding-right: 20px;
+  }
+
+  .oppia-footer-info-icon {
+    font-size: 20px;
+    margin-right: 5px;
+  }
+
+  .oppia-lesson-info {
+    cursor: pointer;
+    display: flex;
+    list-style-type: none;
+    padding-left: 2px;
+    padding-top: 10px;
+  }
+
+  .oppia-lesson-info-h1 {
+    color: #fff;
+    display: inline;
+    font-size: 17px;
+    font-weight: normal;
+  }
+
+  @media (max-width: 650px) {
+    .oppia-exploration-footer .oppia-exploration-footer-sharing-links ul {
+      margin: 0 -8px;
+    }
+  }
+
+  .oppia-exploration-footer .oppia-exploration-footer-sharing-links li {
+    float: left;
+  }
+
+  .oppia-exploration-footer .oppia-navbar-footer-info-icon {
+    height: 56px;
+    padding: 12px;
+  }
+
+  .oppia-exploration-footer .oppia-navbar-footer-info-icon:hover {
+    background: #fff;
+    color: #009688;
+    height: 56px;
+    padding: 12px;
+  }
+
+  @media (max-width: 475px) {
+    .oppia-exploration-footer-sharing-links {
+      display: none;
+    }
+  }
+
+  @media (max-width: 658px) {
+    .oppia-exploration-footer .oppia-share-exploration-footer {
+      display: none;
+    }
+    .oppia-exploration-footer .oppia-navbar-footer-info-icon {
+      margin-top: -3px;
+    }
+    .oppia-exploration-footer .oppia-navbar-footer-info-icon:hover {
+      margin-top: -3px;
+    }
+  }
+
+  @media (min-width: 768px) {
+    .oppia-exploration-footer .oppia-exploration-footer-info-icon {
+      display: none;
+    }
+  }
+</style>
 <!-- TODO(#14625): Incorporate the CSS file inclusion in webpack. -->
```

Comment:
I see the .css files being called here for adding css, can we move the css style from this html file to the component css file.

Reply:
Done.